browser/css/jssidebar.css

```diff
@@ -93,9 +93,18 @@
 }
 
 .sidebar.ui-expander-content {
+	width: 309px;
```

Comment:
is that needed?

Reply:
yes the 309px is needed, cause otherwise the sidebar elements jump. Maybe we don't need the 330px for the complete sidebar width. The idea was to fix the existing bug and than check if we can reduce stuff.

Reply:
| **Issue at Master** | **Fix** |
| ------------- | ------------- |
|![2022-06-24 11_45_38-Sidebar fixed with and no jumping frame in frame no additional padding by Andrea](https://user-images.githubusercontent.com/8517736/175510736-fa895df7-7b2d-4027-be3d-5f939d8b3746.png)|![2022-06-24 11_49_33-Sidebar fixed with and no jumping frame in frame no additional padding by Andrea](https://user-images.githubusercontent.com/8517736/175510749-d6e1a96b-2f6b-43da-9d4f-fbe852aa279f.png)|
| This is the existing sidebar layout if you are at writer when tab selected | with the defined with of 309px |